web/templates/edited_notice.hbs

```diff
@@ -1,4 +1,9 @@
+<div id="editing_notifications" class="edit_notifications"></div>
+{{#if is_typing}}
+    {{> editing_notifications typing=true}}
+{{/if}}
```

Comment:
In the above code, the `is_typing` is used to check if the message is being edited when you are rendering that particular message_container, and display the notification accordingly. Once the typing is done, this is hidden until you re render that container. 

The `id=editing_notifications` div is used for showing typing notifications, when you already have the `message_container` rendered (basically that message is within your view).